python/samples/concepts/plugins/crew_ai_plugin.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+
+from semantic_kernel import Kernel
+from semantic_kernel.core_plugins.crew_ai import CrewAIEnterprise
+from semantic_kernel.core_plugins.crew_ai.crew_ai_models import InputMetadata
```

Comment:
this should be exposed at the crew_ai level as well:
```suggestion
from semantic_kernel.core_plugins.crew_ai import InputMetadata
```